src/chainer.test.js

```diff
@@ -1 +1,41 @@
+'use strict';
+
 const { chainer } = require('./chainer');
+
+describe('chainer', () => {
+
+  it('should chain the unary functions correctly', () => {
+    const mockFunctionA = (x) => x + 1;
+    const mockFunctionB = (x) => x * 2;
+    const mockFunctionC = (x) => x - 3;
+    const mockFunctionD = (x) => x / 4;
+
+    const chain
```

Comment:
There is unnecessary space and new line between 'const chain' and 'chainer'. Consider keeping the initialization of the variable on one line for readability.